spring-cloud-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/endpoint/NacosDiscoveryEndpointAutoConfiguration.java

```diff
@@ -42,4 +42,10 @@ public NacosDiscoveryEndpoint nacosDiscoveryEndpoint(
 		return new NacosDiscoveryEndpoint(nacosDiscoveryProperties);
 	}
 
+	@Bean
+	public NacosDiscoveryHealthIndicator nacosDiscoveryHealthIndicator(
```

Comment:
ConditionalOnEnabledHealthIndicator  annotation condition this method

user can disable this HealthIndicator with: 
```
management:
  health:
    nacosDiscovery:
      enabled: false
```